src/Lunr/Vortex/APNS/APNSPayload.php

```diff
@@ -31,7 +29,7 @@ public function __construct()
     {
         $this->elements = [];
 
-        $this->elements['priority'] = APNSPriority::HIGH;
+        $this->elements['priority'] = APNSPriority::High;
```

Comment:
Are we sure we don't want to set the value in the array?

Reply:
What's the benefit of _not_ using the enum?